support-frontend/app/models/GeoData.scala

```diff
@@ -0,0 +1,19 @@
+package models
+
+import com.gu.i18n.{Country, CountryGroup}
+
+case class GeoData(countryCode: Option[String], stateCode: Option[String]) {
+  def countryGroup: Option[CountryGroup] = countryCode.flatMap(CountryGroup.byFastlyCountryCode)
+  def country: Option[Country] = countryCode.flatMap(CountryGroup.countryByCode)
+  def validatedStateNameForCountry: Option[String] = for {
+    c <- country
+    r <- stateCode
+    stateName <- c.statesByCode.get(r) orElse c.statesByCode.values.find(_ == r)
+  } yield stateName
+  def validatedStateCodeForCountry: Option[String] = for {
+    c <- country
+    r <- stateCode
+    stateName <- c.statesByCode.keySet.find(_ == r) orElse c.statesByCode.map(_.swap).get(r)
```

Comment:
why do we need to support lookups against name as well?

Reply:
Ah, good spot, I may have got confused with the Australia states that use the code as the name, but that's in the def now. I'll do some more AU testing to confirm I can remove this.

Reply:
I've removed this